internal/util/update/update_test.go

```diff
@@ -678,6 +678,46 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
 	}
 }
 
+// TestCommand_Run_failInvalidRepo verifies Run fails if the repo is invalid
+func TestCommand_Run_failInvalidRepo(t *testing.T) {
+	for _, strategy := range kptfilev1alpha2.UpdateStrategies {
+		t.Run(string(strategy), func(t *testing.T) {
+			g := &testutil.TestSetupManager{
+				T: t,
+				ReposChanges: map[string][]testutil.Content{
+					testutil.Upstream: {
+						{
+							Data:   testutil.Dataset1,
```

Comment:
We are trying to move more towards the `pkgbuilder` pattern, as using golden data sets becomes difficult when there are multiple repos where each has multiple commits. But for this test the content doesn't really matter, so it is ok here.

Reply:
I'll start basing other tests on that pattern.